### PR TITLE
Bugfix/mobile ux design pass

### PR DESCRIPTION
**Pull request by KhalidAdan**
This PR comes out of a conversation Alla and I had where she noticed / wanted some extra design changes. Each commit explains what was added and why, but the `Tooltip`, `ResultsTable` component and the `ProgressBar` need some extra love on mobile.

### CODE REVIEW

File: components/Hooks/index.tsx [hunk 1]
```diff
@@ -64,3 +64,29 @@ export function useStore() {
...
+    // Check on mount (callback is not called until a change occurs)
+    if (media.matches) {
+      setTargetReached(true)
+    }
+
+    return () => media.removeListener(updateTarget)
+  }, [])
+
+  return targetReached
+}
```
Comment (KhalidAdan) [inline components/Hooks/index.tsx:94]: reusable Nextjs compatible media query hook to get responsive breakpoints in JS

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
image
The Legal Status dropdown is now on top of the Marital Status one, see screenshot

**Comment by JeremyKennedy** [comment]
Also, it appears that the Legal Status progress item is checked before the user enters any input (after income).

**Comment by KhalidAdan** [comment]
> Also, it appears that the Legal Status progress item is checked before the user enters any input (after income).

the progress bar changes should fix this one, and now that's it's merged in I'll rebase and confirm

Edit: yup the merge fixed it!

**Review by JeremyKennedy — APPROVED** [review]
Both issues (legal status progress and dropdown overlap) appear resolved. Approved!

**Pull request merged**